dashboard-app/src/dashboard/ByVolunteer/datePickerConstraints.ts

```diff
@@ -0,0 +1,82 @@
+import moment from 'moment';
+import { DateRangePickerConstraint } from '../../components/DatePicker/types';
+
+// Date rules:
```

Comment:
Is this the same as ByTime constraints? Would it make more sense to have `DayDatePickerConstraints` and `MonthDatePickerConstraints`?

Reply:
They are the same. Depends on where you land on the whole premature abstraction thing and permitting duplication to allow for future divergence. I'm easy 🤷🏼‍♂️

Reply:
Let's add a comment/issue as a reminder for future abstraction

Reply:
See #110